Source/ChidoriMenu.swift

```diff
@@ -7,34 +7,34 @@
 
 import UIKit
 
-class ChidoriMenu: UIViewController {
-    let tableView: UITableView = UITableView(frame: .zero, style: .plain)
+public class ChidoriMenu: UIViewController {
+    public let tableView: UITableView = UITableView(frame: .zero, style: .plain)
```

Comment:
I would expect this so be internal (implementation detail)

Reply:
Allow me to disagree, some of these properties should stay public to allow for easier theming/customization

Reply:
Totally fine.